src/test/java/unit/in/ac/bits/protocolanalyzer/utils/BeautifyTest.java

```diff
@@ -0,0 +1,37 @@
+package unit.in.ac.bits.protocolanalyzer.utils;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import in.ac.bits.protocolanalyzer.utils.Beautify;
+
+public class BeautifyTest {	
+	@Rule
+	public ExpectedException expectedEx = ExpectedException.none();
+	@Test
+	public void testBeautify() {
+		byte bytes1[] = {0 , 16 , 2 , 15 , 4};
+		String mode = "ip4";
+		assertThat(Beautify.beautify(bytes1, mode), equalTo("INVALID-ADDRESS"));
+		byte bytes2[] = {-65 , 66 , -67 , 68 };
+		assertThat(Beautify.beautify(bytes2, mode), equalTo("191.66.189.68"));
```

Comment:
please use the assertThat(java.lang.String reason, T actual, Matcher<? super T> matcher). The reason indicates the fail condition. For the above assert statement, the fail condition could be phrased as "IPv4 address beautification fails."